lib/core/janitor.js

```diff
@@ -50,12 +50,12 @@ class Janitor {
    * @param {object} securities
    * @returns {Promise}
    */
-  async loadSecurities (securities = {}) {
+  async loadSecurities (securities = {}, userId = '-1') {
```

Comment:
This parameter is not currently used, as `this.createSecurity` still takes only 2 arguments.

Also, I'm not sure about this default value. The "-1" kuid is reserved for the anonymous user, i.e. a non-authenticated user invoking the API. I'm not sure that this should be the default value when loadSecurities is invoked by Kuzzle itself upon startup.

Last remark: you should probably receive a nullable User object instead of a user kuid, for the same reason as the one stated in one of my previous remarks: if `admin:loadSecurities` is executed by a plugin, you won't get any user object at all, so you cannot have a valid user id.